.gitignore

```diff
@@ -114,4 +114,9 @@ run/
 /bukkit/build/
 /api/build/
 /.gradle/
-.gradle/
\ No newline at end of file
+.gradle/
+
+# Why weren't these in the .gitignore?
+/build/
+/common/build/
+/plugin/build/
```

Comment:
Not sure actually! I think it might be because they're in my local ignore file. I thought */build/ should catch all build paths, though, no?

Reply:
In this project, it catches some but doesn't catch others, .gitignore can be pretty weird sometimes